doc/source/faq.rst

```diff
@@ -43,16 +43,28 @@ Similarly, if you have many workers processing large chunks of data you may
 be using much more memory than you expect. If you limit the number of workers
 *and* the size of the data chunks being processed by each worker you can
 reduce the overall memory usage. Default chunk size can be configured in Satpy
-by setting the following environment variable:
+by setting using the following around your code:
 
-.. code-block:: bash
+.. code-block:: python
+
+    with dask.config.set("array.chunk-size": "32MiB"):
+      # your code here
+
+The value of the chunk-size can be given in different ways, see here:
+https://docs.dask.org/en/stable/api.html#dask.utils.parse_bytes
+There are other ways to set dask configuration items, including configuration
+files or environment variables, see here:
+https://docs.dask.org/en/stable/configuration.html
```

Comment:
Would it be worth it to point out that a basic `dask.config.set(...)` at the top of their code would also accomplish this without having to use a context manager?

Reply:
Good idea, but here again, I feel it should be its own separate section.

Reply:
Section added, please have a look.